egs/iwslt22_ta/ST/local/transcript_cleaning.py

```
@@ -1,20 +1,21 @@
 #!/usr/bin/env python
 # Copyright 2020 Kanari AI (Amir Hussein)
 # Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+import argparse
+import os
 import pdb
```

Ruff (0.15.18)

[error] Import for `pdb` found

(T100)